src/main/java/com/puppycrawl/tools/checkstyle/checks/modifier/ClassMemberImpliedModifierCheck.java

```diff
@@ -27,18 +27,18 @@
 
 /**
  * <p>
- * Checks for implicit modifiers on nested types in classes.
+ * Checks for implicit modifiers on nested types in classes and records.
```

Comment:
if we treated enums and "nested type" I think we can treat record as "nested type" also.

Reply:
https://github.com/checkstyle/checkstyle/pull/8578#discussion_r472288927